veniq/baselines/semi/recommend.py

```diff
@@ -40,13 +41,15 @@ def _get_method_subtree(class_decl: List[str]) -> AST:
         javalang_ast = build_ast(_name)
         ast = AST.build_from_javalang(javalang_ast)
         os.unlink(_name)
-    except JavaSyntaxError as e:
+    except JavaSyntaxError:
         os.unlink(_name)
-        raise e
+        raise WrongInputToApi("Javalang failed to build AST tree (javalang.JavaSyntaxError)")
```

Comment:
We need to correct the class name `WrongInputToApi`. Here we know nothing about API (if you meant web API). It may confuse readers

Reply:
can u suggest something? "WrongInput" is OK?

Reply:
For example, https://docs.python.org/3/library/exceptions.html#ValueError